src/hubinit/repulishjob.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/config;
+import ballerina/log;
+import ballerina/runtime;
+import ballerina/websub;
+import mosip/repository;
+import mosip/services;
+
+public type RepublishJob object {
+
+    public function __init() {
+    }
+  
+    service republishservice = service {
+
+        resource function onTrigger(services:HubServiceImpl hubServiceImpl, websub:Hub webSubHub) {
+            repository:RestartRepublishContentModel[] unsentMessages = hubServiceImpl.getUnsentMessages(config:getAsString("mosip.hub.restart_republish_time_offset"));
```

Comment:
Who will be updating mosip.hub.restart_republish_time_offset ?

Reply:
i have discussed this with puneet and in current scenerio we get offset from props but in kube it automatically restarts so i have talked to puneet he will be finding a way to get republish time and we can read it from there.currently reading from properties whoever deploys has to change as of now.

Reply:
done updated config after each job